### PR TITLE
Tw quiz

### PR DESCRIPTION
**Pull request by Thomaswaters05**
Finished.. I needed to add " " instead of "". Ready for review.

PREV POST: (Currently cannot figure out the math for the valid form. I can get the characters to appear + 1 and grow. But i cannot get the tree to follow that path to create that shape. Still working on this but wanted to PR so you could see where I was sitting with my quiz).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JS QUIZ</title>
+</head>
+<body>
+
```
Comment (copypastedeveloper) [inline index.html:8]: INDENTATION IS IMPORTANT!!!!!!

File: quiz.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    characterOption = characterOption.value;
+    generateTreeObject(height, characterOption);
+        };
+}
+
+
+// TW PERSONAL NOTE: Creating the object and deleted my prior wrong doing of height * charoption (<<<--not correct). Create an object, console.log it, and create the next function.
+// think of it as information traveling from one block to the next!! (TW-Study more on this)
+
+function generateTreeObject (height, characterOption) {
```
Comment (copypastedeveloper) [inline quiz.js]: Can you eliminate this function?  As a rule of thumb, if your function is just calling another function, it's probably extraneous.

File: quiz.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+// TW NOTE: TAKE NOTE OF SPACING BETWEEN AND USE README!!
+
+function growTree (tree) {
+   var characters = tree.ch;
+   for (var i = 0; i < tree.ht; i++) {
+    var numChar = 2*i+1;
+    var numSpace = tree.ht-i;
+    console.log(" ".repeat(numSpace) + characters.repeat(numChar));
+  }
+  }
```
Comment (copypastedeveloper) [inline quiz.js]: INDENTATION IS IMPORTANT!!!!!!!!

File: quiz.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  if (e.keyCode == 13){
+   correctInput()
+  }
+}
+
+// ****TESTING AND ADDING COMMENTS IN CASE SOMEONE DOES NOT INPUT PROPERLY**** PERSONAL NOTE: This came easy!
+
+function correctInput () {
+  if (height.value >= 100) {
+    alert("Please enter value lower than 100. You may choose any character you please for the next input field.");
```
Comment (copypastedeveloper) [inline quiz.js]: What if the user doesn't select a character?

**Comment by copypastedeveloper** [comment]
@Thomaswaters05 a couple of things:

## Requirements
- [x] Fix the indentation issues in the html and js files.  
- [x] Add validation for missing character selection

Everything else looks very nice.

**Comment by Thomaswaters05** [comment]
Nathan: Ready for re-check please.

Thank you sir! I have fixed the indentation issues and added a validation for both the missing numerical selection and character selection.

**Comment by copypastedeveloper** [comment]
@Thomaswaters05 looks good. 👍

**Comment by Thomaswaters05** [comment]
Thank you!!

**Pull request merged**